tests/integration/components/x-select-actions-test.js

```diff
@@ -75,13 +75,13 @@ describe('Integration: XSelectActionsComponent', function() {
           {{#xs.option value="Hello"}}Hello{{/xs.option}}
         {{/x-select}}
       `);
-        Ember.warn = sinon.spy();
+        warn = sinon.spy();
```

Comment:
@rwjblue I had pinged you about this a couple months ago in the ember-cli-babel repo but lost track of it while on leave.  With this change here things blow up because `warn` is undefined.  Is there a better way to do this with the new modules api?

Reply:
correction, the error is:

> "warn" is read-only